client/src/views/Simulation/components/RunButton.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="btn-group">
+    <button
+      type="button"
+      class="btn btn-primary"
+      :class="{ 'blue' : !autoRunInput }"
+      @click="onClickRun"
+    >
+      <font-awesome-icon :icon="['fas', ( autoRunInput ? 'pause' : 'play') ]" />
```

Comment:
I like this idea, quite nice, however, we should probably preserve the same blue when the button is in auto-run mode. Currently, it switches to gray so I'd do : `class="btn btn-primary blue"`